packages/icons/src/components/ChevronUp.tsx

```diff
@@ -0,0 +1,30 @@
+/** This file is auto-generated, do not edit it manually **/
```

Comment:
Where did you get that icon? Is it really auto-generated?))0)

Since I cannot find it in figma icon-pack